test/fixtures/api/responses/get-balance-sheet.json

```diff
@@ -0,0 +1,68 @@
+{
+  "balances": [
+    {
+      "counterparty": "rKm4uWpg9tfwbVSeATv4KxDe6mpE9yPkgJ",
+      "balances": [
```

Comment:
You can use `patternProperties` instead of `properties`. I'm kind of on the fence about which option is better; it's nice to have the explicit notation that the address is the counterparty in the second option, but it's also nice to have the lower redundancy in the first option. If you like the first option better, then we can go with that.